### ISSUE TITLE
pop pk plot for >1 PopulationSimulationSets

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite.ReportingEngine#174]
In the example attached there are 2 populations sets: one with the set name "10mg" and another one with the set name "25 mg". 
[Test06.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/4563907/Test06.zip)
<details><summary>PopulationSimulationSets definition</summary>

```
popSimSet1 <- PopulationSimulationSet$new(
	simulationFile = file.path(pkmlDir, "Raltegravir 10 mg   (lactose formulation).pkml"), 
	populationFile = file.path(inputDir, "Raltegravir Adult Population.csv"), 
	pathID = "Organism|Heart|Plasma|Raltegravir|Concentration", 
	pathName = "Raltegravir Heart", 
	pathUnit = "µg/l",
        pkParameters = c("C_max", "t_max"), 
	pkParametersUnits = c("µg/l", "h"),
	simulationSetName = "10mg"
)
popSimSet2 <- PopulationSimulationSet$new(
	simulationFile = file.path(pkmlDir, "Raltegravir 25 mg  (lactose formulation).pkml"), 
	populationFile = file.path(inputDir, "Raltegravir Adult Population.csv"), 
	pathID = "Organism|Heart|Plasma|Raltegravir|Concentration", 
	pathName = "Raltegravir Heart", 
	pathUnit = "µg/l",
        pkParameters = c("C_max", "t_max"), 
	pkParametersUnits = c("µg/l", "h"),
	simulationSetName = "25 mg"
)
```
</details>

In this case the expectation for pk parameters box-whisker plots is: 
_for each combination of {PK-Parameter; Output} one plot **with 2 boxes** is created (**one box per PopulationSimulationSet**}_

However there is only 1 box per plot created:
![AUC_inf](https://user-images.githubusercontent.com/25061876/80802434-96936080-8baf-11ea-8b2f-042df7e222f1.png)
But it should be something like this:
![grafik](https://user-images.githubusercontent.com/25061876/80802725-70ba8b80-8bb0-11ea-80ad-d76961f800b9.png)
(s. also Matlab-Example reports, e.g. https://github.com/Open-Systems-Pharmacology/Reporting-Engine/blob/develop/advanced_examples/03_Population_Pediatric_Raltegavir/figures/pKParameter_Appendix.docx chapter 1.5)

Seems that pk values are grouped by the _population name_ (which in this example is the same for both PopulationSimulationSets) and not by PopulationSimulationSets

**Comment by Yuri05** [comment]
s. also https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/175

**Comment by pchelle** [comment]
@Yuri05 here is the update using the new Output class

A few remarks: 
- Output class does not currently handle the selection of pkParameters, 
I will do that in the next PR to fix issue #144 for pk parameter plots on both mean and population workflows, if that's okay.
- So, currently it is only possible to select pk parameters using the task option yParameters (in attached file). There is also an option xParameters to plot yParameters as a function of xParameters.
- As requested simulationSetName need to be unique, so you get directly an error if that's not the case while you try to initialize a workflow
- If you try pediatric or ratioComparison, you will also need one unique reference population.
So, you need to define for one of the PopulationSimulationSets, the following input `referencePopulation = TRUE` otherwise, you'll get an error while initializing the workflow.
[Test06.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/4581016/Test06.zip)

**Comment by Yuri05** [comment]
@pchelle Great!

Regarding the question:
>Currently the selection of PK parameters to plot is handled that way,
Let me know if you prefer this to be included in Output
`popWorkFlow$plotPKParameters$yParameters <- c("C_max", "t_max")`

Yes, let's include PK-Parameters in the output definition!
